src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -180,15 +180,17 @@ const ImagesTable = ({
         />
         <Toolbar>
           <ToolbarContent>
-            <ToolbarItem>
-              <Link
-                to={resolveRelPath('imagewizard')}
-                className="pf-c-button pf-m-primary"
-                data-testid="create-image-action"
-              >
-                Create image
-              </Link>
-            </ToolbarItem>
+            {!experimentalFlag && (
```

Comment:
I'm starting to feel like `experimentalFlag` should get renamed to something else, like `blueprintFlag` but out of scope for sure :smile: